main.cpp

```diff
@@ -1 +1,238 @@
+if (guesses >= 8) {cout << "\n / \\";}
+if (guesses >= 9) {cout << "\n ___";}
+if (guesses == 10) {cout << "\n|   |\nYOU LOST!";}
+}
+
+void printoptions() { //print options
+    cout << "\nSelect One The Following Option:" <<
+            "\n[1] Guess a letter" <<
+            "\n[2] Guess the word" <<
+            "\n[3] Check guessed letters" <<
+            "\n[4] View options again" <<
+            "\n[5] Quit game" << endl <<
+            "[6] Check Current Word (FOR TESTING ONLY)\n";
+}
+
+void printword(string word, vector<char> cg) { // function to print
+    int i;
+    for (i = 0; i < word.length(); ++i) {
+        if (count(cg.begin(), cg.end(), word[i])) {
+        std::cout << word[i] << " ";
+    }
+    else
+    cout << "_ ";
+    }
+
+}
+
+
+int main(){
+    char input{}; //for user's letter guesses
+    int choice{}; //for user choice
+    string wordinput{}; //for when user wants to guess the entire word
+    vector<char> guessedchar{}; //array for all the letters the user have guessed
+    vector<string> list{"special", "random", "roast", "firetruck", "throw", "trend", "blast", "gasoline"};
+    vector<char> correctguesses{}; //letters that were guessed correctly
+    int guesses{}; // number of guesses by the user
+    
+    srand (time(NULL));
+    int random = rand() % list.size(); //randomise the word
+    string word = list[random];
+    
+    
+    printoptions();
+
+    while (guesses <= 10) { 
+        cout << "Remaining Guesses: " << 10 - guesses << endl;
+        print(guesses); //print hangman
+        cout << endl;
+
+        if (guesses == 10) { //if user has lost the game
+            cout << "\nMission Failed. We'll get them next time." << endl;
+            cout << "\nThe word is " << word << "." << endl;
+            cout << "Would you like to play again? (Y/N)" << endl;
+                        input = ' '; //resets the char 'input'
+                        while (input != 'y' || input != 'Y') {
+                            cin >> input;
+                            if (input == 'y' || input == 'Y') 
+                            {
+                                guesses = 0;
+                                guessedchar.clear();
+                                srand (time(NULL));
+                                int random = rand() % list.size(); //randomise the word
+                                string word = list[random-1];
+                                printoptions();
+                                break;
+                            }
+                            else if (input == 'n' || input == 'N')
+                                return 0;
+                            else {
+                                cout << "Invalid option please type either 'y' or 'n'" << endl;
+                    }
+                }
+        }
+
+        else if (guesses < 10) //if user still has guesses remaining
+        cout << "The word is " << word.length() << " characters long." << endl;
```

Comment:
I assume you know how it works when there's an if statement with no brackets, so you should use indentation to make it clearer